plugin/opam.vim

```diff
@@ -69,9 +69,9 @@ function! s:Opam(bang,...) abort
 endfunction
 
 function! s:Complete(A,L,P)
-  let installed = split((system("opam switch -s -i 2> /dev/null")), "\n")
-  call map(installed, 'opam#chomp(v:val)')
-  return join(installed, "\n")
+  let switches = split((system("opam switch --short 2> /dev/null")), "\n")
```

Comment:
`-i` i no more available, is it required to have only installed ones?

Reply:
i don't think so